plotly/plotly/plotly.py

```diff
@@ -50,7 +50,7 @@
 DEFAULT_PLOT_OPTIONS = {
     'filename': "plot from API",
     'fileopt': "new",
-    'world_readable': True,
+    'world_readable': False,
```

Comment:
@theengineear - My thought is to include saving the `world_readable` as a "required" step in the new getting-started instructions and not changing the default behaviour. So, everyone who is new will save the default to False, and be aware of privacy, and we don't introduce the backwards incompatibility for everyone else who is upgrading.

thoughts?